docs/content/preview/drivers-orms/java/_index.md

```diff
@@ -27,10 +27,10 @@ The following projects can be used to implement Java applications using the Yuga
 
 | Projects | Documentation and Guides | Example Apps |
 | ------- | ------------------------ | ------------ |
-| Ebean ORM | [Documentation](ebean/)<br /> [Hello World App](../../develop/build-apps/java/ysql-ebean/)<br /> [Blog](https://blog.yugabyte.com/ebean-orm-yugabytedb/)| [Ebean ORM App](https://github.com/yugabyte/orm-examples/tree/master/java/ebean/)
-| Hibernate ORM | [Documentation](hibernate/)<br />[Hello World App](../../develop/build-apps/java/ysql-hibernate/)<br />[Blog](https://blog.yugabyte.com/run-the-rest-version-of-spring-petclinic-with-angular-and-distributed-sql-on-gke/)<br /> | [Hibernate ORM App](https://github.com/yugabyte/orm-examples/tree/master/java/hibernate/)
-| Spring Data YugabyteDB | [Documentation](../../integrations/spring-framework/sdyb/)<br />[Hello World App](../../develop/build-apps/java/ysql-sdyb/)<br />[Blog](https://blog.yugabyte.com/spring-data-yugabytedb-getting-started/) | [Spring Data YugabyteDB Sample App](https://github.com/yugabyte/spring-data-yugabytedb-example/)
-| Spring Data JPA | [Documentation](../../integrations/spring-framework/sd-jpa/)<br />[Hello World App](../../develop/build-apps/java/ysql-spring-data/)<br />[Blog](https://blog.yugabyte.com/run-the-rest-version-of-spring-petclinic-with-angular-and-distributed-sql-on-gke/) | [Spring Data JPA App](https://github.com/yugabyte/orm-examples/tree/master/java/spring/)
+| Ebean ORM | [Documentation](ebean/)<br /> [Hello World](../orms/ysql-ebean/)<br /> [Blog](https://blog.yugabyte.com/ebean-orm-yugabytedb/)| [Ebean ORM App](https://github.com/yugabyte/orm-examples/tree/master/java/ebean/)
+| Hibernate ORM | [Documentation](hibernate/)<br />[Hello World](../orms/ysql-hibernate/)<br />[Blog](https://blog.yugabyte.com/run-the-rest-version-of-spring-petclinic-with-angular-and-distributed-sql-on-gke/)<br /> | [Hibernate ORM](https://github.com/yugabyte/orm-examples/tree/master/java/hibernate/)
+| Spring Data YugabyteDB | [Documentation](../../integrations/spring-framework/sdyb/)<br />[Hello World](../../develop/build-apps/java/ysql-sdyb/)<br />[Blog](https://blog.yugabyte.com/spring-data-yugabytedb-getting-started/) | [Spring Data YugabyteDB Sample App](https://github.com/yugabyte/spring-data-yugabytedb-example/)
```

Comment:
Why is this hello world still in build apps?

Reply:
Yes this is one I have mentioned in the description. This one is not an ORM. I need to combine it with its integrations page.